### ISSUE TITLE
Zoom overflow

### ISSUE THREAD

**Issue by ooker777** [soscripted/sox#371]
After installing updating to version 2.3.11DEV the text is overflown in zoomed screen:

![](https://i.imgur.com/VyH2c4W.gif)

 - Installed Version: 2.3.11DEV  Environment: Tampermonkey
 - [SOX Errors logged to the browser console](https://pastebin.com/ALtbR9b7)

 - Features Enabled 

    ["Appearance-addAuthorNameToInboxNotifications","Appearance-alignBadgesByClass","Appearance-answerTagsSearch","Appearance-colorAnswerer","Appearance-highlightQuestions","Appearance-localTimestamps","Appearance-markEmployees","Appearance-metaChatBlogStackExchangeButton","Appearance-spoilerTip","Appearance-standOutDupeCloseMigrated","Appearance-tabularReviewerStats","Appearance-topAnswers","Appearance-unspoil","Appearance-hideWelcomeBackMessage","Comments-autoShowCommentImages","Comments-commentReplies","Comments-commentShortcuts","Comments-confirmNavigateAway","Comments-copyCommentsLink","Comments-moveBounty","Comments-showCommentScores","Comments-hiddenCommentsIndicator","Editing-addSBSBtn","Editing-editComment","Editing-editReasonTooltip","Editing-findAndReplace","Editing-kbdAndBullets","Editing-titleEditDiff","Editing-inlineEditorEverywhere","Editing-pasteImagesDirectly","Flags-flagOutcomeTime","Flags-flagPercentages","Flags-flagPercentageBar","Sidebar-hideCommunityBulletin","Sidebar-hideJustHotMetaPosts","Sidebar-hideHireMe","Sidebar-hideLoveThisSite","Sidebar-hideHotNetworkQuestions","Sidebar-linkedToFrom","Sidebar-hotNetworkQuestionsFiltering","Chat-chatEasyAccess","Chat-replyToOwnChatMessages","Chat-renameChat","Extras-alwaysShowImageUploadLinkBox","Extras-linkedPostsInline","Extras-parseCrossSiteLinks","Extras-quickAuthorInfo","Extras-shareLinksMarkdown","Extras-sortByBountyAmount","Extras-copyCode","Extras-dailyReviewBar","Extras-openLinksInNewTab","Extras-showQuestionStateInSuggestedEditReviewQueue"]

**Comment by shu8** [comment]
That's weird :/ does it only happen on that site? Does it also only happen
when viewing a question or does it happen on all pages (eg the front page,
etc..).

I'll hopefully get it fixed by this weekend!

**Comment by ooker777** [comment]
It seems like the SBS button authors this bug. If the feature is disabled then it's back to normal.

# What happens when it's enabled?

Then there are sites that have this bug and sites that don't. At first I suspect that the new design (the navigation buttons on the left bar) makes this, so I also note whether a site has it or not (nd vs od):

Sites that have this bug:
- Interpersonal Skills (nd)
- Academia (nd)
- Mathematics (nd)
- ELU (od)

Sites that don't:
- Parenting (nd)
- Unix & Linux (nd)
- ELL (od)
- Linguistics (nd)

The bug doesn't affect in help pages or homes. If it happens in a main site, then it will be the same in the its meta.

**Comment by ooker777** [comment]
It seems like the SBS button authors this bug. If the feature is disabled then it's back to normal.

What happens when it's enabled? Then there are sites that have this bug and sites that don't. At first I suspect that the new design (the navigation buttons on the left bar) makes this, so I also note whether a site has it or not (nd vs od):

Sites that have this bug:
- Interpersonal Skills (nd)
- Academia (nd)
- Mathematics (nd)
- ELU (od)

Sites that don't:
- Parenting (nd)
- Unix & Linux (nd)
- ELL (od)
- Linguistics (nd)

The bug doesn't affect in help pages or homes. If it happens in a main site, then it will be the same in the its meta.

**Comment by shu8** [comment]
Thank you! I will try looking at this on the weekend!

**Comment by shu8** [comment]
@ooker777 just to clarify, when you say it doesn't happen if SBS is disabled, do you mean when SBS is toggled off (but the feature is still enabled in the cogs settings panel), or when it's entirely disabled from the settings panel?

If it's happening when the feature is toggled on then I think this is expected just because of the way the feature works! It needs a large space on the sides so may overflow if SBS mode is enabled.

**Comment by ooker777** [comment]
yeah, it was because the SBS mode is toggled on and enable in the setting. The trouble is that it's automatically toggled on in every question. This is fine for non-zoom-users but not for me. 

The inconsistency I list above seems to be that the upper sites I visited other users' questions, while the lower sites I visited my own (which don't open the answer box by default). Because zooming will make the typing box almost fill the screen width, in the end I just turn it off.
